docs/mondoo.mdx

```diff
@@ -0,0 +1,91 @@
+---
+description: The Mondoo packer provisioner will scan builds for vulnerabilities and misconfigurations by executing security policies-as-code.
```

Comment:
@scottford-io Please have a look at this description, I just threw something in quickly, feel free to improve!

Reply:
```suggestion
description: The Mondoo packer provisioner scans machine-image builds for vulnerabilities and misconfigurations by executing security policies-as-code.
```

@preslavgerchev small suggestion